osquery/tables/system/windows/system_info.cpp

```diff
@@ -36,12 +36,16 @@ QueryData genSystemInfo(QueryContext& context) {
   }
 
   WmiRequest wmiSystemReq("select * from Win32_ComputerSystem");
+  WmiRequest wmiSystemReqProc("select * from Win32_Processor");
   std::vector<WmiResultItem>& wmiResults = wmiSystemReq.results();
+  std::vector<WmiResultItem>& wmiResultsProc = wmiSystemReqProc.results();
   if (wmiResults.size() != 0) {
     long numProcs = 0;
     wmiResults[0].GetLong("NumberOfLogicalProcessors", numProcs);
     r["cpu_logical_cores"] = INTEGER(numProcs);
     wmiResults[0].GetLong("NumberOfProcessors", numProcs);
+    r["cpu_physical_processors"] = INTEGER(numProcs);
```

Comment:
Why are you adding in a new column? If you do this we lose compatibility with the other tables.. Is there any reason that this value is different than the `cpu_physical_cores`? If you add this here can you modify the other table schemas also?